### DISCUSSION TITLE
Add an extra file to a PR/MR

### DISCUSSION THREAD

**Discussion by giany** [renovatebot/renovate#25505] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

In my repo I have a file that contains:

```
locals {
  version           = "0.0.5"
}
```

I use renovatebot to update some helm charts versions and I was thinking if there a way to autoincrement that specific version every time there is a new helm chart available and add that file to the same MR?

### What you tried so far.

I've looked at allowedPostUpgradeCommands param but it does not do what I want.

**Comment by rarkins** [comment]
You need to use postUpgradeTasks

**Comment by giany** [comment]
I assume you refer to postUpgradeTasks, right? Would this end up in config.ts or renovate.json? Hmm

**Reply by rarkins** [reply]
It can be in either, but is usually a per-repo setting so therefore renovate.json

**Reply by giany** [reply]
This seems to work but I have to craft some Bash script to do this. Is there a better approach to detect if a MR is a patch/minor/major release and then based on that to use some tool to increment my version accordingly?

**Reply by rarkins** [reply]
No, never
